tests/github/w3c/webtransport/index.html

```diff
@@ -4,9 +4,10 @@
   <meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport">
   <title>WebTransport</title>
   <meta content="w3c/ED" name="w3c-status">
-  <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-ED" rel="stylesheet">
-  <link href="https://www.w3.org/2008/site/images/favicon.ico" rel="icon">
+  <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-ED" rel="stylesheet" type="text/css">
+  <meta content="Bikeshed version b94c7e755, updated Fri Feb 19 16:28:59 2021 -0800" name="generator">
```

Comment:
It looks like you regen'd this test by just running `bikeshed` on it, and with a version that's a few months old as well. Could you update your Bikeshed and run `bikeshed test --rebase github/w3c/webtransport/index.bs` instead? As it is, the test will fail in CI.